tutorbackup/templates/backup/build/backup/restore_services.sh

```diff
@@ -2,5 +2,6 @@
 set -e
 tar -xvf backup/backup.tar.xz
 mysql --password=$MYSQL_ROOT_PASSWORD --user=$MYSQL_ROOT_USERNAME --host=$MYSQL_HOST --port=$MYSQL_PORT < data/mysql_dump.sql
-mongorestore --host=$MONGODB_HOST --port=$MONGODB_PORT --stopOnError /data/mongodb_dump/
+mongorestore --host=$MONGODB_HOST --port=$MONGODB_PORT \
+  {% if MONGODB_USERNAME and MONGODB_PASSWORD %}--username=$MONGODB_USERNAME --password=$MONGODB_PASSWORD{% endif %}--stopOnError /data/mongodb_dump/
```

Comment:
Missing whitespace: if the credentials are set, this results in `--password=foo--stopOnError`.

Reply:
Nice catch